frontend/test/metabase/scenarios/question/new.cy.spec.js

```diff
@@ -149,6 +149,29 @@ describe("scenarios > question > new", () => {
         cy.findByText("Second collection").should("not.exist");
       });
     });
+
+    it("should be possible to create a question based on a question in another user personal collection", () => {
+      cy.signOut();
+      cy.signIn("nocollection");
+      startNewQuestion();
+      popover().findByText("Orders").click();
+      visualize();
+      saveQuestion("Personal question");
```

Comment:
Should we maybe do this via an API request to speed up things?

Reply:
➕